packaging/helm/trento-server/charts/trento-web/templates/deployment.yaml

```diff
@@ -43,6 +43,15 @@ spec:
             - "{{ .Values.webService.port }}"
             - --collector-port
             - "{{ .Values.collectorService.port }}"
+            {{ if .Values.mTLS.enabled }}
+            - --enable-mtls
+            {{ end }}
+            - --cert
```

Comment:
shouldn't `--cert` `--key` and `--ca` be inside the `if` ? :thinking:

Reply:
doesn't change much because if the flag is not set they are being ignored but i will change it for clarity, thanks!

Reply:
True, I remember the implementation.
Yet, my OCD thanks you :joy:

Reply:
no you are right it is clearer, i just didn't think of it